runtime/query.go

```diff
@@ -320,6 +322,17 @@ func parseMessage(msgDescriptor protoreflect.MessageDescriptor, value string) (p
 		fm := &field_mask.FieldMask{}
 		fm.Paths = append(fm.Paths, strings.Split(value, ",")...)
 		msg = fm
+	case "google.protobuf.Value":
+		var x interface{}
+		err := json.Unmarshal([]byte(value), &x)
+		if err != nil {
+			return protoreflect.Value{}, err
+		}
+		v, err := structpb.NewValue(x)
+		if err != nil {
+			return protoreflect.Value{}, err
+		}
+		msg = v
```

Comment:
I think we should use `protojson`, not the standard library marshaler. Also, can't we just do this in one fell swoop?
```suggestion
		var v structpb.Value
		err := protojson.Unmarshal([]byte(value), &v)
		if err != nil {
			return protoreflect.Value{}, err
		}
		msg = &v
```